src/ui/components/ReactPanel.tsx

```diff
@@ -383,118 +390,42 @@ function ReactQueuedRenderListItem({
   );
 }
 
-type CommittedRenderProps = CommonProps & {
-  commitPoint: TimeStampedPoint;
-};
-
-function ReactCommittedRenderListItem({
```

Comment:
:speech_balloon: FWIW I still _really_ think we ought to be showing commits somehow :)

Reply:
Agreed - would just like a different UI treatment for it. This is where having a figma brainstorming session helps